src/check.ts

```diff
@@ -59,6 +59,21 @@ export interface CheckRequest {
   externalContent?: ExternalContent;
 }
 
+export interface LiveSearchRequest {
+  'request-id': string;
```

Comment:
Is this hypen needed because of platform interface?

Reply:
yes. See https://gitlab.com/acrolinx/core-platform/reuse-service/-/blob/main/doc/rest-endpoints.md#extended-phrase-prefix-search